mocha/mocha-node.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for mocha 2.2.5
```

Comment:
Making the comment more specific sounds fine, but that kind of convention should be standardized across DefinitelyTyped and therefore self-evident from the file names.